projects/storefrontlib/shared/components/ng-select-a11y/ng-select-a11y.directive.ts

```diff
@@ -85,11 +107,37 @@ export class NgSelectA11yDirective implements AfterViewInit {
           options.forEach(
             (option: HTMLOptionElement, index: string | number) => {
               const ariaLabel = `${option.innerText}, ${+index + 1} ${translation} ${options.length}`;
-              this.renderer.setAttribute(option, 'aria-label', ariaLabel);
+              this.renderer.setAttribute(
+                option,
+                this.ariaLabelAttribute,
+                ariaLabel
+              );
             }
           );
         });
     }
     observerInstance.disconnect();
   }
+
+  appendValueToAriaLabel(
```

Comment:
Could we add some jsdoc comment to explain what this method is doing? We should also cover it with unit tests.

Reply:
Added comment + unit test